python/paddle/distributed/collective.py

```diff
@@ -188,12 +188,13 @@ def new_group(ranks=None, backend=None, timeout=_default_timeout):
     Examples:
         .. code-block:: python
 
-            import paddle
+            >>> import paddle
```

Comment:
```suggestion
            >>> # doctest: +REQUIRES(env: DISTRIBUTED)
            >>> import paddle
```

<img width="969" alt="image" src="https://github.com/PaddlePaddle/Paddle/assets/38436475/09ceb272-bc70-40c1-8623-c96f916b0220">